.github/workflows/tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - ${{ github.event.repository.default_branch }}
+  pull_request:
+    branches:
+      - ${{ github.event.repository.default_branch }}
+    paths:
+      - '*.js'
+      - 'package.json'
+
+jobs:
+  tests:
+    env:
+      NPM_CONFIG_PROGRESS: "false"
```

Comment:
This might be a thing in node v6.. I'm not sure.. but npm for a while has defaulted progress to false if it's running in a known CI..   so I _assume_ this is no longer necessary.. but node 6 might be memory-holed for me.

Reply:
Yes, it's off by default in CI: https://docs.npmjs.com/cli/v6/using-npm/config#progress.

Though it probably doesn't hurt to turn it off explicitly. Out of curiosity, why does it need to be off, anyway?

Reply:
It's just a fraction slower to show the progress bar and nobody is watching the progress on CI anyway

Reply:
Also it's off for npm v6.. but I dunno what version of npm is loaded for node 6?

Reply:
It is npm v3 for node 6